.github/workflows/full-build.yml

```diff
@@ -218,22 +217,16 @@ jobs:
           install_name_tool -change @rpath/librsvg-2.2.dylib $(brew --prefix librsvg)/lib/librsvg-2.2.dylib $(brew --prefix librsvg)/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader_svg.so
           codesign --sign - --force --preserve-metadata=entitlements,requirements,flags,runtime $(brew --prefix librsvg)/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader_svg.so
           GDK_PIXBUF_MODULEDIR=$(brew --prefix)/lib/gdk-pixbuf-2.0/2.10.0/loaders gdk-pixbuf-query-loaders --update-cache
-      - name: Ensure Homebrew Library Path (ARM only)
-        if: matrix.target.arch == 'arm'
-        run: echo "DYLD_LIBRARY_PATH=/opt/homebrew/lib" >> $GITHUB_ENV
+      - name: Ensure Homebrew Library Path
+        run: echo "DYLD_LIBRARY_PATH=$(brew --prefix)/lib" >> $GITHUB_ENV
         shell: bash
-      - name: Set up Python
```

Comment:
Is relying on the homebrew installed Python going to consistently work?

Reply:
It's the same Python version I use locally. I do not think the python installation used was the problem, so we can revert this change, if you think that's better.

Reply:
Ok great. Ya, I think being consistent with using setup-python to get the right Python version would be better.